README.md

```diff
@@ -35,6 +35,17 @@ In case you need sudo:
 curl -fsSL https://raw.githubusercontent.com/kool-dev/kool/master/install.sh | sudo bash
 ```
 
+For Windows
+
+Download the installer [here](https://github.com/kool-dev/kool/releases)
+
+Or execute the bat with admin
+
+```bash
+bitsadmin /transfer KoolDevInstallDownload /dynamic /download /priority foreground https://raw.githubusercontent.com/kool-dev/kool/master/install.bat install-kool-dev.bat
```

Comment:
Is this bat helpful/necessary at some situation? I mean, having the installer seems like the best/simpler option for Windows native users.

Reply:
No more, the problem with this is the way, in each new installation the path is added, and is not easy to uninstall.

The installer solve this problem.